components/chainhook-sdk/src/indexer/stacks/mod.rs

```diff
@@ -432,6 +462,29 @@ pub fn standardize_stacks_block(
             pox_cycle_length: pox_cycle_length.try_into().unwrap(),
             confirm_microblock_identifier,
             stacks_block_hash: block.block_hash.clone(),
+
+            block_time: block.block_time,
+            // TODO: decode `signer_bitvec` into an easy to use bit string representation (e.g. "01010101")
+            signer_bitvec: block.signer_bitvec.clone(),
```

Comment:
@rafaelcr we get these from `/new_block` in consesus-serialize format, e.g. `0x000800000001ff`. Is there existing code I can call to decode this? We should deliver it in the payloads in regular bitfield format (a string of `1` and/or `0`).

Reply:
Yes, I believe so, it's probably included in the update we did to the `stacks-codec` here https://github.com/hirosystems/clarinet/pull/1524